api/envoy/extensions/http/cache/file_system_http_cache/v3/file_system_http_cache.proto

```diff
@@ -77,4 +74,46 @@ message FileSystemHttpCacheConfig {
   //
   // [#not-implemented-hide:]
   uint32 cache_subdivisions = 6;
+
+  // The amount of the maximum cache size or count to evict when cache eviction is
+  // triggered. For example, if ``max_cache_size_bytes`` is 10000000 and ``evict_fraction``
+  // is 0.2, then when the cache exceeds 10MB, entries will be evicted until the cache size is
+  // less than or equal to 8MB.
+  //
+  // The default value of 0 means when the cache exceeds 10MB, entries will be evicted only
+  // until the cache is less than or equal to 10MB.
+  //
+  // Evicting a larger fraction will mean the eviction thread will run less often (sparing
+  // CPU load) at the cost of more cache misses due to the extra evicted entries.
+  //
+  // [#not-implemented-hide:]
+  float evict_fraction = 7;
+
+  // The longest amount of time to wait before running a cache eviction pass. An eviction
+  // pass may not necessarily remove any files, but it will update the cache state to match
+  // the on-disk state. This can be important if multiple instances are accessing the same
+  // cache in parallel.
+  //
+  // If an eviction pass has not happened within this duration, the eviction thread will
+  // be awoken and perform an eviction.
+  //
+  // If unset, there will be no eviction passes except those triggered by cache limits.
+  //
+  // [#not-implemented-hide:]
+  google.protobuf.Duration max_eviction_frequency = 8;
```

Comment:
Technically it's a period, the reciprocal is frequency.

Reply:
Good call. Updated. (Though "every 10 minutes" is a frequency, there's no way to phrase that as a duration and variable name.)